types/dockerode/index.d.ts

```diff
@@ -454,6 +454,29 @@ declare namespace Dockerode {
     }
     /* tslint:enable:interface-name */
 
+    interface CreateVolumeOptions {
+        Name?: string;
+        Driver?: string;
+        DriverOpts?: { [key: string]: string } | undefined;
+        Labels?: { [label: string]: string };
+    }
+
+    interface VolumeCreateResponse {
+        Name: string;
+        Driver: string;
+        Mountpoint: string;
+        CreatedAt?: string | undefined;
+        Status?: string;
```

Comment:
Based on the linked docs, it looks like `Status` is a map of key/value pairs and not a string

```suggestion
Status?: { [key: string]: string } | undefined;
```